packages/core/contracts/merkle-distributor/implementation/MerkleDistributor.sol

```diff
@@ -248,6 +238,8 @@ contract MerkleDistributor is Ownable {
 
     // Verify claim is valid and mark it as completed in this contract.
     function _verifyAndMarkClaimed(Claim memory _claim) private {
+        // Check claimed rewards do not exceed remaining funding for the window.
+        require(_claim.amount <= merkleWindows[_claim.windowIndex].remainingAmount, "Insufficient funding for window");
```

Comment:
I dont think this is strictly required as the call on line 175 would revert if there was not enough but it's better to have a more descriptive revert message than just `underflow`

Reply:
Agree. We should drop this as it's an unnecessary gas expenditure.